app/themes/light/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,7 @@
 <div id="header-wrapper">
 			<%= nav_bar brand: image_tag("#{AppSettings['design.header_logo']}", width: 40, class: "pull-left logo") + "#{AppSettings['settings.product_name'] + " " if AppSettings['settings.product_name']}" "#{AppSettings['settings.site_name']}", responsive: true do %>
 					<%= menu_group pull: :right do %>
-							<span class="hidden-lg hidden-sm hidden-md"><%= render('result/search_form') %></span>
+							<span class="hidden-lg hidden-sm hidden-md hidden-xs"><%= render('result/search_form') %></span>
 							<%= menu_divider %>
```

Comment:
The light and flat theme does not actually use this code because the menu is included in the main layout files `light.html.erb` and `flat.html.erb`.  I checked at it does not seem to have the same formatting issue.